compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/declaration/FirOverrideChecker.kt

```diff
@@ -34,11 +34,7 @@ object FirOverrideChecker : FirClassChecker() {
             stubTypesEqualToAnything = false
         )
 
-        val firTypeScope = declaration.unsubstitutedScope(
-            context.sessionHolder.session,
-            context.sessionHolder.scopeSession,
-            withForcedTypeCalculator = true
```

Comment:
Note that this is not necessary as mentioned at: https://github.com/JetBrains/kotlin/pull/4124#discussion_r583631765